### PR TITLE
fix: removed duplicated "Get Started" content

### LINKED ISSUE #2863: Getting started page is duplicated
The content of the "Getting Started" page is duplicated under _content/en/docs/install-setup/_index.htm_ and _content/en/docs/getting-started/_index.html_.

### PR DESCRIPTION
**Pull request by hferentschik**
- moved "Get Started" to the /docs to simplify redirects
- added dedicated _index.md ti install-setup with clarification that
  these sections go beyond the simple "Get Started"
- aligning heading style for pages in install-setup
- setting status of security-features to draft (content not up tp date)
- aliging naming style in community menu for documentation -> "Contribute Documentation"

fixes #2863

### CODE REVIEW

File: content/en/community/documentation/_index.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 ---
 title:
-linktitle: Writing Documentation
+linktitle: Contribute Documentation
```
Comment (deanesmith) [inline content/en/community/documentation/_index.md:3]: Should this be "Contributing Documentation"?

**Comment by hferentschik** [comment]
@jha-cloudbees, I would love to know why your approvals don't merge the PR. I know we had issues with that which were caused by a Lighthouse issue, but I thought that was resolved. For all I can tell you are in the OWNERS file as reviewer and approver. @abayer WDYT?

**Pull request merged**